### DISCUSSION TITLE
Examples of extended reports?

### DISCUSSION THREAD

**Discussion by jwhite66** [kimai/kimai#2604] [Questions & Answers]
The documentation suggests here:
  https://www.kimai.org/documentation/reporting.html#extending-the-reports
that it's possible to write code to extend or add new reports.

But I have not yet found any examples.  Did I understand the documentation correctly?  If so, anyone have pointers to an example of how to create custom reports?

**Comment by kevinpapst** [comment]
This is only an example how you can add custom links to the report screen.
I will add an example to the DemoBundle. 
The link you mentioned refers to an EventSubscriber like the existing ones:
https://github.com/Keleo/DemoBundle/tree/master/EventSubscriber
